third_party/3/maxminddb/compat.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for maxminddb.compat (Python 3)
+
+from ipaddress import IPv4Address, IPv6Address
+
+from typing import Any, Union
+
+def compat_ip_address(address: Any) -> Union[IPv4Address, IPv6Address]: ...
```

Comment:
This should use the same signature as `ipaddress.ip_address()`: It should return `Any`. Otherwise the user is forced to use `isinstance` checks on the return value even if they know the type returned. It should also take an `object` as argument, since the value passed in is ultimately converted to a string using `str()`, which can be applied to all objects.